pandas/core/indexes/base.py

```diff
@@ -1121,13 +1120,13 @@ def _summary(self, name=None):
             tail = self[-1]
             if hasattr(tail, "format") and not isinstance(tail, str):
                 tail = tail.format()
-            index_summary = ", %s to %s" % (pprint_thing(head), pprint_thing(tail))
+            index_summary = f", {pprint_thing(head)} to {pprint_thing(tail)}"
```

Comment:
pprint_thing may be unnecessary here now that we're py3-only.  can you take a look